gh_release/ECHAM5_model.xml

```diff
@@ -158,43 +158,1051 @@
+        <attrdefs>Fang et al 2004</attrdefs>
+        <attrdomv>
+          <rdom>
+            <rdommin>NA</rdommin>
+            <rdommax>NA</rdommax>
+            <attrunit>linear distance (m)</attrunit>
+          </rdom>
+        </attrdomv>
+      </attr>
+      <attr>
+        <attrlabl>vol_29_100</attrlabl>
+        <attrdef>lethal habitat </attrdef>
+        <attrdefs>Fang et al 2004</attrdefs>
+        <attrdomv>
+          <rdom>
+            <rdommin>NA</rdommin>
+            <rdommax>NA</rdommax>
+            <attrunit>volume (m^3*1000)</attrunit>
+          </rdom>
+        </attrdomv>
+      </attr>
+      <attr>
+        <attrlabl>days_30_31</attrlabl>
+        <attrdef>adult optimum days</attrdef>
+        <attrdefs>Wismer and Christie 1987</attrdefs>
+        <attrdomv>
+          <rdom>
+            <rdommin>NA</rdommin>
+            <rdommax>NA</rdommax>
+            <attrunit>days</attrunit>
+          </rdom>
+        </attrdomv>
+      </attr>
+      <attr>
+        <attrlabl>height_30_31</attrlabl>
+        <attrdef>adult optimum meters</attrdef>
+        <attrdefs>Wismer and Christie 1987</attrdefs>
+        <attrdomv>
+          <rdom>
+            <rdommin>NA</rdommin>
+            <rdommax>NA</rdommax>
+            <attrunit>linear distance (m)</attrunit>
+          </rdom>
+        </attrdomv>
+      </attr>
+      <attr>
+        <attrlabl>vol_30_31</attrlabl>
+        <attrdef>adult optimum volume</attrdef>
+        <attrdefs>Wismer and Christie 1987</attrdefs>
+        <attrdomv>
+          <rdom>
+            <rdommin>NA</rdommin>
+            <rdommax>NA</rdommax>
+            <attrunit>volume (m^3*1000)</attrunit>
+          </rdom>
+        </attrdomv>
+      </attr>
+      <attr>
+        <attrlabl>durStrat</attrlabl>
+        <attrdef>Duration of stratified period</attrdef>
+        <attrdefs>de Stasio et al 1996</attrdefs>
+        <attrdomv>
+          <rdom>
+            <rdommin>NA</rdommin>
+            <rdommax>NA</rdommax>
+            <attrunit>days</attrunit>
+          </rdom>
+        </attrdomv>
+      </attr>
+      <attr>
+        <attrlabl>winter_dur_0-4</attrlabl>
+        <attrdef>winter duration</attrdef>
+        <attrdefs>Jones et al 2006, Hokanson 1977</attrdefs>
+        <attrdomv>
+          <rdom>
+            <rdommin>NA</rdommin>
+            <rdommax>NA</rdommax>
+            <attrunit>days</attrunit>
+          </rdom>
+        </attrdomv>
+      </attr>
+      <attr>
+        <attrlabl>spring_days_in_10.5_15.5</attrlabl>
+        <attrdef>Duration of optimal incubation period</attrdef>
+        <attrdefs>Wismer and Christie 1987</attrdefs>
+        <attrdomv>
+          <rdom>
+            <rdommin>NA</rdommin>
+            <rdommax>NA</rdommax>
+            <attrunit>days before June 1 (in spring only)</attrunit>
+          </rdom>
+        </attrdomv>
+      </attr>
+      <attr>
+        <attrlabl>mean_surf_jul</attrlabl>
+        <attrdef>summer warmth index </attrdef>
```

Comment:
Should the attrdef's maybe be the longer-form description in that file? These versions are pretty terse.